tests/components/sunsynkweb/test_coordinator.py

```diff
@@ -0,0 +1,70 @@
+"""Basic coordinator tests."""
+
```

Comment:
the conversation is still marked as resolved, so I'm adding it again: You are still testing the coordinator

Reply:
these tests cover only the coordinator to the extent that if we get an error accessing the information, the sensors' state are reflected correctly. The comments and file names are misleading, will update